### ISSUE TITLE
LD_LIBRARY_PATH isseuse after following installation instructions

### ISSUE THREAD

**Issue by jhaye** [xldenis/creusot#529]
After following the installation instructions for Creusot, when calling `cargo creusot` I get the following error:
```
error while loading shared libraries: librustc_driver-0deccb1aea69cdba.so: cannot open shared object file: No such file or directory
```

On my machine this shared library is located at `~/.rustup/toolchains/nightly-2022-06-21-x86_64-unknown-linux-gnu/lib`. This path however is not in my `LD_LIBRARY_PATH`, so I have to do the following `LD_LIBRARY_PATH=$LD_LIBRARY_PATH:~/.rustup/toolchains/nightly-2022-06-21-x86_64-unknown-linux-gnu/lib cargo creusot` to make it work.

I could just update the env var and be done with it, but I wonder, is this a problem with my installation, or is something missing from the installation instructions?

**Comment by gmorenz** [comment]
I ran into the same issue. The problem appears to have been that I was using a different version of rustc than creusot. Copying creusot's `rust-toolchain` file into the root of the project I was trying creusot on fixed it.

**Comment by xldenis** [comment]
Yes, that is mandatory for all verification tools unfortunately. I'll add this to the documentation.
